core/pom.xml

```diff
@@ -356,6 +356,11 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-test-tags_${scala.binary.version}</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Can you declare this dependency in the root pom and just reference it here, like it's done for others?

That way there's only one place where the version is defined, if/when other modules need to use it.